packages/@yoda/wifi/index.js

```diff
@@ -102,7 +102,7 @@ NetworkListener.prototype._onevent = function onevent (data) {
        * @type {boolean}
        */
       this.emit('stateupdate', this._online)
-    } else if (this._online === true && msg['Wifi'] === false) {
+    } else if (this._online === true && msg['Network'] === false) {
```

Comment:
Compared with ahead code, why need change `msg['Wifi']` to `msg['Network']`?
The upper code shows: `if (msg['Network'] === true)` then set `this._online` as `true`.

Reply:
@yonzkon Please try to answer @morrowind's question :(